engines/sci/engine/script_patches.cpp

```diff
@@ -6502,7 +6501,7 @@ static const uint16 pq3PatchGiveLocketPoints[] = {
 	0x7a,                                // push2
 	0x38, PATCH_UINT16(0x00ff),          // pushi 0x00ff - using last flag slot, seems to be unused
 	0x39, 0x0a,                          // pushi 10d - 10 points
-	0x45, 0x06, 0x04,                    // callb export000_6, 4
+	0x45, 0x06, 0x04,                    // callb 4 proc0_6
```

Comment:
I used this opcode as it appears in SCI Viewer.
I don't see a reason to change those at all.
In fact it gets more confusing for me, because I'm still using SCI Viewer almost all the time when looking at script code.